### ISSUE TITLE
Default simulator?

### ISSUE THREAD

**Issue by shams-ahmed** [skyweb07/Snap.swift#51]
I'm new to snapshot testing, are there are recommending/guidelines to follow when testing?

I have this strange behaviour when I create images for iPhone x it passes and when I try to use another simulator it fails. from the diffs, I can see that the UI is slightly higher by about 4/5 px.

**Comment by skyweb07** [comment]
yeah this is because of phone density, depends on the iphone you run the tests it will look for the @2x, @3x image if exported, I'm working on something to fix this and make it device agnostic
